### PR TITLE
chore(codegen): add preview modules to release profile

### PR DESCRIPTION
**Pull request by zhumin8**
Prepare to merge this when ready to release. 
Alter title to reflect what's expected to publish in release notes.
ref: [release please commits](https://github.com/googleapis/release-please#how-should-i-write-my-commits)

-----
Update (@emmileaf) : release-please override for release note message will be added as part of PR description for #1517.
Merge order: #1517 > #1469 > This PR.

### CODE REVIEW

**Review by emmileaf — APPROVED** [review]

**Review by meltsufin — APPROVED** [review]

**Comment by emmileaf** [comment]
Will merge this by hand after #1517 and #1469.

**Comment by emmileaf** [comment]
This is currently blocked on javadoc issues, addressing in https://github.com/googleapis/gapic-generator-java/pull/1277.
```
Error:  /home/runner/work/spring-cloud-gcp/spring-cloud-gcp/spring-cloud-previews/google-cloud-accessapproval-spring-starter/src/main/java/com/google/cloud/accessapproval/v1/spring/AccessApprovalSpringAutoConfiguration.java:47: error: reference not found
Error:   * Auto-configuration for {@link AccessApprovalClient}.
```
The correct class here (used in generated code) should be `AccessApprovalAdminClient`. There are 8 exceptions similar to this one.

**Pull request merged**
